packages/calypso-e2e/src/lib/blocks/block-flows/youtube.ts

```diff
@@ -10,7 +10,7 @@ const selectors = {
 	embedUrlInput: `${ blockParentSelector } input`,
 	embedButton: `${ blockParentSelector } button:has-text("Embed")`,
 	editorYouTubeIframe: 'iframe[title="Embedded content from www.youtube.com"]',
-	publishedYouTubeIframe: `iframe.youtube-player`,
+	publishedYouTubeIframe: 'iframe[title="Getting started on @wordpressdotcom"]',
```

Comment:
Still getting an error `waiting for frameLocator('iframe[title="Getting started on @wordpressdotcom"]').locator('text=Getting Started on @wordpress-com').first() to be visible`

Here is how the frontend looks like 

![CleanShot 2024-09-18 at 12 51 21](https://github.com/user-attachments/assets/7c1c54cb-ea24-4836-889b-de467932b021)